src/main/resources/modules/CMS130v10.json

```diff
@@ -0,0 +1,281 @@
+        },
+        {
+          "transition": "Check_For_Procedure",
+          "distribution": 0.1
+        }
+      ]
+    },
+    "Qualifying_Encounter": {
+      "type": "Encounter",
+      "telemedicine_possibility": "none",
+      "name": "Qualifying_Encounter",
+      "direct_transition": "Encounter_Occurred",
+      "codes": [
+        {
+          "system": "SNOMED-CT",
+          "code": 444971000124105,
+          "display": "Annual Wellness Visit",
+          "value_set": "2.16.840.1.113883.3.526.3.1240"
+        },
+        {
+          "system": "SNOMED-CT",
+          "code": 185460008,
+          "display": "Home Healthcare Services",
+          "value_set": "2.16.840.1.113883.3.464.1003.101.12.1016"
+        },
+        {
+          "system": "SNOMED-CT",
+          "code": 185463005,
+          "display": "Office Visit",
+          "value_set": "2.16.840.1.113883.3.464.1003.101.12.1001"
+        },
+        {
+          "system": "CPT",
+          "code": "99395",
+          "display": "Preventive Care Services - Established Office Visit, 18 and Up",
+          "value_set": "2.16.840.1.113883.3.464.1003.101.12.1025"
+        },
+        {
+          "system": "CPT",
+          "code": 99385,
+          "display": "Preventive Care Services-Initial Office Visit, 18 and Up",
+          "value_set": "2.16.840.1.113883.3.464.1003.101.12.1023"
+        }
+      ]
+    },
+    "Encounter_Occurred": {
+      "type": "SetAttribute",
+      "attribute": "encounter_occurred",
+      "name": "Encounter_Occurred",
+      "value": true,
+      "direct_transition": "End_Encounter"
+    },
+    "Telehealth_Visit": {
+      "type": "Encounter",
+      "encounter_class": "ambulatory",
+      "reason": "",
+      "telemedicine_possibility": "none",
+      "codes": [
+        {
+          "system": "CPT",
+          "code": "98970",
+          "display": "Virtual Encounter",
+          "value_set": "2.16.840.1.113883.3.464.1003.101.12.1089"
+        },
+        {
+          "system": "SNOMED-CT",
+          "code": 185317003,
+          "display": "Telephone Visit",
+          "value_set": "2.16.840.1.113883.3.464.1003.101.12.1080"
+        }
+      ],
+      "name": "Telehealth_Visit",
+      "direct_transition": "Encounter_Occurred"
+    },
+    "Colonoscopy_Procedure": {
+      "type": "Procedure",
+      "codes": [
+        {
+          "system": "SNOMED-CT",
+          "code": 446521004,
+          "display": "Colonoscopy",
+          "value_set": "2.16.840.1.113883.3.464.1003.108.12.1020"
+        },
+        {
+          "system": "LOINC",
+          "code": "12503-9",
+          "display": "Fecal Occult Blood Test",
+          "value_set": "2.16.840.1.113883.3.464.1003.198.12.1011"
+        },
+        {
+          "system": "SNOMED-CT",
+          "code": 396226005,
+          "display": "Flexible Sigmoidoscopy",
+          "value_set": "2.16.840.1.113883.3.464.1003.198.12.1010"
+        },
+        {
+          "system": "LOINC",
```

Comment:
I'm not sure if this actually matters for the measure evaluatation, some of these do look like Procedures but some seem like they should be Observations (the ones with LOINC codes). For example I see in the measure criteria `[["Laboratory Test, Performed": "FIT DNA"]](https://ecqi.healthit.gov/mcw/2022/ecqm-dataelement/laboratorytestperformedfitdna.html)` . If a Procedure resource can meet that criteria then no problem, but it seems like it may have to be an Observation instead.